docs/integrations/logfire.md

```diff
@@ -75,4 +82,40 @@ Now start your DBOS application. You should see your logs and traces appear on t
 
 ![DBOS Logs and Traces on Logfire](./assets/logfire-screenshot.png)
 
+
+## Configure DBOS with Logfire SDK
+
+If you're using the Logfire SDK (currently only support Python), set the `LOGFIRE_TOKEN` environment variable to your write token:
+
+```shell
+export LOGFIRE_TOKEN='your-write-token'
+```
+
+:::tip
+
+If you're deploying your app on DBOS Cloud, make sure to set `LOGFIRE_TOKEN` in your application's [environment variables](../production/dbos-cloud/secrets).
+
+:::
+
+Then, configure your DBOS application to use Logfire and enable OpenTelemetry traces. You don't need to set `otlp_traces_endpoints` or `otlp_logs_endpoints`, because Logfire SDK already automatically sets up exporters.
```

Comment:
```suggestion
Then, configure your DBOS application to use Logfire and enable OpenTelemetry traces. You don't need to set `otlp_traces_endpoints` or `otlp_logs_endpoints`, because the Logfire SDK automatically configures exporters.
```